### ISSUE TITLE
Kinect data not showing on Apple Silicone M1 macOS 11.5.2

### ISSUE THREAD

**Issue by alexgfaria** [shiffman/OpenKinect-for-Processing#165]
It seems like the data is getting through, but it isn't presented on the screen. I've tried adding frameRate(30) and removing fullScreen(P3D); I tried with version 4 betas, 3, and 2, but nothing works.

Please help!

**Comment by hakimmazouz** [comment]
Did you find a way?

**Comment by alexgfaria** [comment]
> Did you find a way?

yes. I installed processing 4.0b1 and processing-java through there. depth data now works, except live feed video.

**Comment by ksenia-5** [comment]
**Apple Silicone M1 macOS 11.5.2 with Processing 4.0b8 in Java**
I get the error "The constructor Kinect(Kinect) is undefined", when running code from the RGB depth test found here
https://github.com/shiffman/OpenKinect-for-Processing/blob/master/OpenKinect-Processing/examples/Kinect_v1/RGBDepthTest/RGBDepthTest.pde

Any solution?
